rpcs3/rpcs3_app.cpp

```diff
@@ -4,6 +4,9 @@
 
 #include "rpcs3qt/welcome_dialog.h"
 
+#include "discord_rpc.h"
```

Comment:
you should #ifdef these includes as well

Reply:
I didn't really see the need to ifdef these, it doesn't really contribute anything to the codebase except 2 extra lines.

Reply:
you're including unnecessary files if you were to have it disabled, i don't see how it's justifiable to keep it this way for the sake of keeping the diff small.

Reply:
Alright, I don't see the point of arguing about this any further, so I'll just go ahead and change it.